net.devolutions.RDM.metainfo.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this file isn't installed either. Nor referenced in the manifest for that purpose.

Reply:
Should be fixed